tensorflow/lite/examples/label_image/label_image.cc

```diff
@@ -126,7 +126,7 @@ TfLiteStatus ReadLabelsFile(const string& file_name,
                             size_t* found_label_count) {
   std::ifstream file(file_name);
   if (!file) {
-    LOG(FATAL) << "Labels file " << file_name << " not found\n";
+    LOG(FATAL) << "Labels file " << file_name << " not found" << std::endl;
```

Comment:
Thanks! Is this only needed here? In that case, please leave a comment why std::endl should be used here for future contributors.

Reply:
Well, it's probably needed everywhere, there log is just std::cerr and it doesn't add newline (or autoflush) at the end of LOG. I believe to compensate for this `"\n"` was added at the end of lines (for example like this 34231a62). However  it just looks the same, `"\n"` doesn't flush buffer. So if program aborts, output may be lost.

Line where `"\n"` was replaced with `<< std::endl;`  is the last LOG label_image does when 'label' file is missing, next thing what it does is [exit(-1)](https://github.com/tensorflow/tensorflow/blob/34231a62a6eb62907cd20f7fbcef966f2c6183cc/tensorflow/lite/examples/label_image/label_image.cc#L352), which doesn't flush buffer either. 

@teijeong , would you prefer all  `"\n"` in the label_image replaced with `<< std::endl;`, so it would be consistent?

Reply:
Thanks for the explanation. If it posses potential erroneous behavior, I think it would be better to change all to `std::endl`. After a quick search, this seems to be only one of the few files that has "\n" at the end.

Reply:
Ok, I've replaced all uses of `"\n"` with `std::endl`, and force pushed to the branch. The sole exception is `display_usage()` where only last `"\n"` was replaced with `std:endl`.

Reply:
@teijeong, is there anything else I could to move this pull request forward?